x-pack/legacy/plugins/siem/public/components/empty_value/empty_value.test.tsx

```diff
@@ -28,8 +28,9 @@ describe('EmptyValue', () => {
     expect(toJson(wrapper)).toMatchSnapshot();
   });
 
-  describe('#getEmptyValue', () =>
-    test('should return an empty value', () => expect(getEmptyValue()).toBe('--')));
+  describe('#getEmptyValue', () => {
+    test('should return an empty value', () => expect(getEmptyValue()).toBe('--'));
+  });
```

Comment:
Sometimes the unit tests fail here and give a nasty error about this section being an implicit return so I went ahead and just fixed it here.